Tribler/Test/Core/Modules/RestApi/base_api_test.py

```diff
@@ -59,9 +59,9 @@ def parse_body(self, body):
         return body
 
     def parse_response(self, response):
-        self.assertEqual(response.code, self.expected_response_code)
         if response.code in (200, 400, 500):
             return readBody(response)
+        self.assertEqual(response.code, self.expected_response_code) #TODO move back
```

Comment:
It's better to add your name after the TODO so we always know who created the TODO.